features/FEATURE_BLE/ble/BLETypes.h

```diff
@@ -379,6 +379,9 @@ typedef byte_array_t<32> public_key_coord_t;
 /** Diffie-Hellman key */
 typedef byte_array_t<32> dhkey_t;
 
+/* counter for signed data writes done by GattClient */
```

Comment:
Could you add the second star at the beginning of the comment ? So it can be recognize bt doxygen.